src/api/db/migrate/20221123163119_change_notifications_primary_and_foreign_keys_to_bigint.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeNotificationsPrimaryAndForeignKeysToBigint < ActiveRecord::Migration[7.0]
+  def up
+    # This migration blocks the table, we are running it during Maintenance Window.
+    safety_assured do
+      change_column :notified_projects, :notification_id, :bigint
```

Comment:
After talking with @eduardoj and testing the migration with real data, we think it's a good opportunity to also change the notified_projects id to bigint. Working on it...